### DISCUSSION TITLE
How to implement sentence case by macros?

### DISCUSSION THREAD

**Discussion by yanedie** [fralau/mkdocs-macros-plugin#180] [Q&A]
In sentence case, lowercase most words in a title or heading. Capitalize only the first word of the title. But I don't know what's the namespace of them - the sub nav items (on the left side) and the table of contents (on the right side). Anyone who can help me? I would appreciate it!

**Comment by fralau** [comment]
I am lacking context. Do you refer to the titles and headings of a page? What is the difficulty you are meeting?

**Reply by yanedie** [reply]
Yes, the headings of a page and the name of nav. The difficulty I met was I didn't know how to express them. Like, if I want to extract the value of the metadata of a page, I can use page.meta.xxx.

**Comment by fralau** [comment]
I have a better idea of you want to do. It's advanced stuff, but it should be doable.

Mkdocs-Macros provides the means to access the properties of the objects, not an in-depth knowledge of them.  

Your question is typical of the use case of a plugin (except that, in this case, you wish to do it with a macro, which can make sense).
Since the people who really know what those objects and properties are, the designers of MkDocs, [you could try asking them](https://github.com/mkdocs/mkdocs/discussions)?
